src/hooks/protections/receive_net_message.cpp

```diff
@@ -51,7 +51,7 @@ namespace big
 					buffer.ReadString(message, 256);
 					buffer.ReadQWord(&unk, 64);
 					buffer.ReadBool(&is_team);
-					LOG(INFO) << "[CHAT] from " << player->get_name() << ": " << message;
+					LOG(INFO) << "[CHAT] from " << player->get_name() << ": " << message << " " << ((is_team) ? "[TEAM]" : "[ALL]");
```

Comment:
Why not pring `is_team` instead of `": "` or just do `"[TEAM]: "` and why add space separately?